modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,67 +12,66 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization (including the Federated ones).
-It is the recommended initial solution pending future corporate investments in enterprise specific OAuth 2.0 offerings.
+The Mule OAuth 2.0 Provider is an alternative from MuleSoft for the OAuth 2.0 provider, and can be used in any API Platform organization, including the Federated ones. The Mule OAuth 2.0 provider is the first-step solution for your OAuth 2.0 offerings.
 
-Supported third party OAuth 2.0 providers:
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) versions 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
```

Comment:
Is there a reason on why not using "v" as a prefix for versions? Eg v4.2.0. It's a standard practice in the industry, although not everyone uses it.

Reply:
You can either use it with each version: v3.x, v4.2.0. Since we are talking about two version here, the use of "versions" works better here in my opinion. Kim?

Reply:
I don't think you need either "version" or "v": take a look at how we present Mule OAuth 2.0. Think about how we commonly refer to updated software: Windows 13, OS 5, and so on.

FWIW, Salesforce uses all three:

For API and product version numbers:
Use _earlier_ instead of _lower_ or _below_, as in **Internet Explorer 6.0** and earlier.
Use _later_ instead of _above_, as in Internet Explorer 7.0 and later.
Include one decimal value for Salesforce API version numbers. Example: This object is available in **API version 35.0** and later.
For the Force.com IDE plug-in for Eclipse, use the format Major Release (**Force.com IDE vXX.X**), where vXX.X is the version number of the Eclipse plug-in. For example: Winter ’14 (Force.com IDE v32.0). The Force.com IDE version number is directly tied to the API version number, but the Force.com IDEhas an off-cycle release schedule.

Reply:
removed "v" and "versions" Done!